addons/hr/wizard/hr_plan_wizard.py

```diff
@@ -1,49 +1,106 @@
+                warning = activity_type.get_responsible_id(employee)['warning']
+                if warning:
+                    warnings.add(warning)
+        return warnings
+
+    @api.depends('employee_ids', 'plan_id')
+    def _compute_warning(self):
+        for wizard in self:
+            warnings = wizard._get_warnings()
+
+            if warnings:
+                warning_display = _('The plan %s cannot be launched: <br><ul>', wizard.plan_id.name)
+                for warning in warnings:
+                    warning_display += '<li>%s</li>' % warning
+                warning_display += '</ul>'
+            else:
+                warning_display = False
+            wizard.warning = warning_display
 
     def _get_activities_to_schedule(self):
         return self.plan_id.plan_activity_type_ids
 
     def action_launch(self):
-        for activity_type in self._get_activities_to_schedule():
-            responsible = activity_type.get_responsible_id(self.employee_id)
-
-            if self.env['hr.employee'].with_user(responsible).check_access_rights('read', raise_exception=False):
-                date_deadline = self.env['mail.activity']._calculate_date_deadline(activity_type.activity_type_id)
-                self.employee_id.activity_schedule(
-                    activity_type_id=activity_type.activity_type_id.id,
-                    summary=activity_type.summary,
-                    note=activity_type.note,
-                    user_id=responsible.id,
-                    date_deadline=date_deadline
-                )
-
-        for plan in self:
-            plan.employee_id.message_post(body=_('The plan %s has been started', plan.plan_id.name))
+        self.ensure_one()
+        for employee in self.employee_ids:
+            body = _('The plan %s has been started', self.plan_id.name)
+            activities = set()
+            for activity_type in self._get_activities_to_schedule():
+                responsible = activity_type.get_responsible_id(employee)['responsible']
+                if self.env['hr.employee'].with_user(responsible).check_access_rights('read', raise_exception=False):
+                    date_deadline = self.env['mail.activity']._calculate_date_deadline(activity_type.activity_type_id)
+                    employee.activity_schedule(
+                        activity_type_id=activity_type.activity_type_id.id,
+                        summary=activity_type.summary,
+                        note=activity_type.note,
+                        user_id=responsible.id,
+                        date_deadline=date_deadline
+                    )
+                    activity = _('%s, assigned to %s, due on the %s', activity_type.summary, responsible.name, date_deadline)
+                    activities.add(activity)
+
+            if activities:
+                body += '<ul>'
+                for activity in activities:
+                    body += _('<li>%s</li>', activity)
```

Comment:
The `activity`es are already translated

```suggestion
                    body += '<li>%s</li>' % activity
```